astropy/io/ascii/ui.py

```diff
@@ -410,6 +419,11 @@ def _guess(table, read_kwargs, format, fast_reader):
     else:
         fast_kwargs = None
 
+    # dictionary arguments are passed by reference per default and might
```

Comment:
Within the main code please maintain code style by capitalizing all comments.  (I'm not so fussy about test code).